src/test/java/com/neo4j/docker/neo4jserver/TestBasic.java

```diff
@@ -20,9 +24,16 @@
 import org.testcontainers.containers.wait.strategy.Wait;
 import org.testcontainers.utility.DockerImageName;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
 import java.time.Duration;
+import java.util.Map;
 import java.util.function.Consumer;
 
+import static java.lang.String.format;
```

Comment:
since you moved your tests out of TestBasic.java can you just check that there aren't any unused imports any more